front/app/containers/Admin/pagesAndMenu/containers/CustomPages/Edit/HeroBanner/index.tsx

```diff
@@ -53,6 +54,13 @@ const EditCustomPageHeroBannerForm = ({
     }
   }, [customPage]);
 
+  // disable form if there's no header image when local settings change
+  useEffect(() => {
+    if (!localSettings?.header_bg) {
+      setFormStatus('disabled');
+    }
+  }, [localSettings]);
+
```

Comment:
I've commented out this code, yet it still disables the button when I remove the header_bg.